src/js/view/components/controls/progressBar.js

```diff
@@ -282,15 +282,15 @@ const ProgressBar = function ($container, api, isAd, metadata) {
 
         if (isAd) {
             api.on(AD_TIME, function (data) {
-                if (data && data.duration && data.position) {
+                if (data && data.duration && typeof data.position === "number") {
```

Comment:
It would probably be a good practice to add such check also for `data.duration`, but I guess it doesn't matter when `duration` is 0.